Maple2.Server.Game/PacketHandlers/ItemUseHandler.cs

```diff
@@ -320,4 +327,16 @@ public class ItemUseHandler : PacketHandler<GameSession> {
 
         session.Config.UpdatePremiumTime(hours);
     }
+    
+    private static void HandleSelectItemBox(GameSession session, IByteReader packet, Item item) {
+        short unknown = packet.ReadShort();
+        int index = packet.ReadShort() - 48;
```

Comment:
what does 48 mean here? should it be a named constant?

Reply:
I have no idea. subtracting 48 gets the correct index. Where does this magic number come from? who knows.